src/sign.js

```diff
@@ -0,0 +1,92 @@
+const email = document.querySelector(".email");
+const emailInput = document.querySelector("#email");
+const password = document.querySelector(".password");
+const passwordInput = document.querySelector("#password");
+const submitBtn = document.querySelector(".loginBtn");
+const eyeOn = document.querySelector(".eyeOn");
+const eyeOff = document.querySelector(".eye-off");
+const eyeBtn = document.querySelector(".eyeBtn");
+
+// 에러 메세지 생성 후 추가 함수
+const createErrMsg = (element, message) => {
+  const lastElement = element.lastElementChild;
+  if (lastElement.tagName === "P") lastElement.remove();
+
+  const errMsgTag = document.createElement("p");
+  errMsgTag.textContent = message;
+  element.appendChild(errMsgTag);
+};
+
+// 에러 메세지 제거 함수
+const removeErrMsg = (element) => {
+  const err_msg = element.querySelector("p");
+  if (err_msg) {
+    err_msg.remove();
+  }
+};
+
+// 이메일 검사 함수
+const emailValidator = ({ target }) => {
+  let emailRegExp = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```

Comment:
변하지 않는 값이라서 const 추천드립니다!
```suggestion
  const emailRegExp = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```